### ISSUE TITLE
Cannot open tab's context menu (e.g. go to next diff) with keyboard

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#8500]
Testing #7854

I tried moving focus to a tab and pressing shift-f10. 

You need to listen for Shift+F10 and bring up the context menu (e.g. tree widget also does it.)

**Comment by bpasero** [comment]
@alexandrudima works for me when using "on screen keyboard" pressing the context menu key:

![image](https://cloud.githubusercontent.com/assets/900690/16452419/af15b642-3e08-11e6-9929-884296ad68a7.png)

I do not know about Shift+F10.

**Comment by bpasero** [comment]
All I have is a context menu event handler on the DIV that is the tab, so I am not sure what else I need to do.

**Comment by bpasero** [comment]
Actually Shift+F10 also works for me, just tried.

**Comment by alexandrudima** [comment]
@bpasero The context menu key works for me too. Under Windows, I think it is expected that Shift+F10 also opens the context menu. Maybe for keyboards without a context menu key?

AFAIK that needs a manual listener. i.e.:
```
{ "key": "shift+f10",             "command": "editor.action.showContextMenu",
                                     "when": "editorTextFocus" },
```

**Comment by bpasero** [comment]
@alexandrudima for me it just works, when I press Shift+F10 on a tab that has focus (with blue outline) I get the menu. Weird that you are not?

**Comment by bpasero** [comment]
Even on my thinkpad without VM and without the context menu key.

**Comment by alexandrudima** [comment]
Yeah, I don't get it, I just tried dev, in case I missed a commit. I don't get contextmenu on Win10. Maybe we can ask @dbaeumer to try.

@dbaeumer If you focus your way to a tab title, does shift-f10 show the contextmenu for you?

**Comment by dbaeumer** [comment]
Yes it does 

![capture](https://cloud.githubusercontent.com/assets/1931590/16480911/f334081a-3ea8-11e6-8948-ba282abd9981.png)

**Comment by alexandrudima** [comment]
@bpasero I can show on my machine
